CMakeLists.txt

```diff
@@ -103,17 +103,7 @@ if(WITH_MOAB)
     set(MOAB_ROOT "${DEPS_ROOT_DIR}")
   endif()
   MESSAGE("-- MOAB Root: ${MOAB_ROOT}")
-  find_package(MOAB)
-  message("-- MOAB Found: ${MOAB_FOUND}")
```

Comment:
I think some kind of message about what is found in MOAB would be useful, unless the MOAB cmake file does that?  Like the version string, perhaps?

Reply:
Ah, I see. For the current latest release, the MOAB config file does not come with version strings, since my PR was recently merged to the develop branch. It will then be available in the master branch. So, we need to wait for a new release of MOAB, like 5.6. For now, I think I can add some message strings to the FindMOAB file.